lib_eio/flow.ml

```diff
@@ -60,8 +60,11 @@ let string_source s = cstruct_source [Cstruct.of_string s]
 class virtual sink = object (_ : <Generic.t; ..>)
   method probe _ = None
   method virtual copy : 'a. (#source as 'a) -> unit
+  method virtual write_from : Cstruct.t -> unit
```

Comment:
"write" on its own is fine here ("write buffer" makes sense in English, whereas "read buffer" doesn't).

I think it would also make sense to expose the lowest-level API here, like we do with `Eio.File.pwrite`. And perhaps take a list like that does? e.g.

```suggestion
  method virtual write : Cstruct.t list -> int
```